nix-script.hs

```diff
@@ -41,6 +41,27 @@ haskell = LangDef "haskell" d r i where
   r script = ("runhaskell" , [script])
   i script = ("ghci"       , [script])
 
+python :: LangDef
+python = LangDef "python" d r i where
+  d pkgs = return $
+    Text.unwords (map ("pythonPackages." <>) pkgs)
+  r script = ("python" , [script])
+  i script = ("python" , ["-i", script])
```

Comment:
Please don't write `return $ Text.unwords $ ...`.  You don't need the return / Text.unwords. `...` should be enough.

Also, you might need to add `python` to the dependencies before returning them, since you need to have the python interpreter available in the shell.